src/filters.rs

```diff
@@ -398,6 +426,47 @@ pub fn escape(input: &Value, args: &[Value]) -> FilterResult {
     }
 }
 
+pub fn remove_first(input: &Value, args: &[Value]) -> FilterResult {
+    match *input {
+        Str(ref x) => {
+            match args.first() {
+                Some(&Str(ref a)) => Ok(Str(x.splitn(2, a).collect())),
+                _ => Err(InvalidArgument(0, "Str expected".to_owned())),
+            }
+        }
+        _ => Err(InvalidType("String expected".to_owned())),
+    }
+}
+
+pub fn remove(input: &Value, args: &[Value]) -> FilterResult {
+    match *input {
+        Str(ref x) => {
+            match args.first() {
+                Some(&Str(ref a)) => Ok(Str(x.replace(a, ""))),
+                _ => Err(InvalidArgument(0, "Str expected".to_owned())),
+            }
+        }
+        _ => Err(InvalidType("String expected".to_owned())),
+    }
+}
+
+pub fn strip_html(input: &Value, _args: &[Value]) -> FilterResult {
+    // regexps taken from https://git.io/vXbgS
+    let matchers = [Regex::new(r"(?is)<script.*?</script>").unwrap(),
+                    Regex::new(r"(?is)<style.*?</style>").unwrap(),
+                    Regex::new(r"(?is)<!--.*?-->").unwrap(),
+                    Regex::new(r"(?is)<.*?>").unwrap()];
```

Comment:
You could use lazy_static to initialize these, see https://github.com/cobalt-org/liquid-rust/blob/master/src/lexer.rs#L21